static/docs/get-started/reusing-data.md

```diff
@@ -0,0 +1,48 @@
+# Reusing data
+
+We've explained how to [upload](/doc/get-started/share-data) and
+[download](/doc/get-started/retrieve-data) <abbr>data artifacts</abbr> (with
+`dvc pull` and `dvc push`) as a way to share them on remote storage. This is
+great for several people to work on the same <abbr>DVC project</abbr> (on
+different machines). But what if we wanted to reuse a dataset from a project in
+a completely different one?
+
+That's why we created `dvc import`! This commands can download data from any DVC
+project hosted on a Git repository. It also tracks the imported data by creating
+an import stage ([DVC-file](/doc/user-guide/dvc-file-format)). Example:
+
+```dvc
+$ dvc import https://github.com/iterative/dataset-registry \
+           get-started/data.xml
+```
+
+> This command is only shown for informational purposes. No need to actually run
+> it in order to continue with this guide.
+
+This downloads `data.xml`, adds it to `.gitignore`, and creates the
+`data.xml.dvc` stage file.
+
+<details>
+
+### Expand to learn about how importing works
```

Comment:
title is a bit misleading? Explanation covers only one specific detail.

Reply:
it also out of place here - should be before get?

Reply:
True. Updated title AND added more useful details about the importing process.